src/app/components/videochat/jh-video-chat.directive.js

```diff
@@ -122,7 +122,11 @@
           }
           // If current one is still speaking, there is no need to change
           if ( !(current && current.getSpeaking()) ) {
```

Comment:
Of course if everybody is disabling the video you will end up with some weird situation. We probably need to have placeholders also in the main feed view to handle that. But right now I would consider that a non relevant corner case.